cloudaux/aws/README.md

```diff
@@ -79,6 +79,16 @@ For GCP support run:
     @iter_account_region('kms', accounts=accounts, regions=['us-east-1'], **conn_details)
     def list_keys(conn=None):
         return conn.list_keys()['Keys']
+        
+    # If you want your role to be read-only, you can assume your role and add the read_only flag to connection details
```

Comment:
Are there additional permissions that the calling role will need?

Reply:
no additional permissions required. I updated the documentation to explain how the permissions are limited.